src/jekyll/usage.md

```diff
@@ -11,10 +11,12 @@ This page contains detailed usage and configuration information for the `sbt-pgp
 
 **If you're using the built-in Bouncy Castle PGP implementation, skip this step.**
 
-The first step towards using the GPG command line tool is to make `sbt-pgp` `gpg`-aware.
+`sbt-pgp` uses `gpg` by default. This can be overriden with:
 
     useGpg := true
```

Comment:
s/true/false/?

Reply:
Oh, sorry about that. I fixed in my branch and I didn't push the change!